csvdiff/nix/packages.nix

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+#
+# SPDX-License-Identifier: Apache-2.0
+_: {
+  perSystem = {
+    pkgs,
+    lib,
+    ...
+  }: let
+    pp = pkgs.python3Packages;
+  in {
+    packages = rec {
```

Comment:
is rec needed?

Reply:
That's needed due to `csvdiff`. Removing would give the following error:
```
       error: undefined variable 'csvdiff'

       at /home/hrosten/projects/ci/own-ci-public/csvdiff/nix/packages.nix:13:17:

           12|     packages = {
           13|       default = csvdiff;
             |                 ^
           14|

```